YellowNotes.Core/Dtos/NoteDto.cs

```diff
@@ -8,6 +8,8 @@ public class NoteDto
     {
         public int NoteId { get; set; }
 
+        public int? CategoryId { get; set; }
```

Comment:
how about multiple categories?

Reply:
We decided that for now, we'll only allow selecting a single category.